<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisScreenSizeChoiceDialog</class>
 <widget class="QDialog" name="KisScreenSizeChoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose screen resolution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="lblTitleMessage">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The driver for your  drawing tablet and Windows do not agree on the size of the screen your tablet is connected to. Please select the right option.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioWintab">
        <property name="text">
         <string notr="true">This is what the tablet thinks: NO_I18N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioQt">
        <property name="text">
         <string notr="true">This is what the computer thinks: NO_I18N</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioManual">
          <property name="text">
           <string>If both are wrong, enter the size manually:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item column="0" row="0" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item column="1" row="0">
           <widget class="QLabel" name="lblWidth">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item column="2" row="0">
           <widget class="QSpinBox" name="intWidth">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item column="3" row="0">
           <widget class="QLabel" name="lblXOffset">
            <property name="text">
             <string>X offset:</string>
            </property>
           </widget>
          </item>
          <item column="4" row="0">
           <widget class="QSpinBox" name="intXOffset">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item column="1" row="1">
           <widget class="QLabel" name="lblHeight">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item column="2" row="1">
           <widget class="QSpinBox" name="intHeight">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item column="3" row="1">
           <widget class="QLabel" name="lblYOffset">
            <property name="text">
             <string>Y offset:</string>
            </property>
           </widget>
          </item>
          <item column="4" row="1">
           <widget class="QSpinBox" name="intYOffset">
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRememberSetting">
        <property name="text">
         <string>Do not ask again (hold Shift to ask again)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KisScreenSizeChoiceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KisScreenSizeChoiceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>